orchagent/orchdaemon.cpp

```diff
@@ -19,6 +19,8 @@ extern sai_object_id_t             gSwitchId;
 PortsOrch *gPortsOrch;
 /* Global variable gFdbOrch declared */
 FdbOrch *gFdbOrch;
+/*Global variable gAclOrch declared*/
+AclOrch *gAclOrch;
```

Comment:
no need to make it global
if PFC needs AclOrch, it can receive it via constructor as all other components do
For example, RouteOrch receives pointer to NeighborOrch:
`new RouteOrch(m_applDb, APP_ROUTE_TABLE_NAME, neigh_orch);`

Pfc is created in lines 129 and 169

Reply:
I think it is ok. gAclOrch is used by PfcWdAclHandler, whose objects are short-living.